overlord/snapstate/handlers.go

```diff
@@ -1978,6 +1978,146 @@ func (m *SnapManager) doPreferAliases(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
+// changeReadyUpToTask returns whether all other change's tasks are Ready.
```

Comment:
(nitpick) Hm, hm, when reading this I was a bit confused - the name reads as if everything "up-to" the task is ready. So I assumed it would stop when reading "me". But its really "changeAllReadyExceptOneTask()" or "changeReadyExceptTask(ignoreTask *state.Task) bool" or something.

Reply:
I made maybe again a too cute linguistic choice. This is complicated, first of all up to can mean both including and excluding depending:

https://english.stackexchange.com/questions/178743/is-up-to-inclusive-or-exclusive

Notice that the function is not checking task itself.  So one longer option would be changeReadyIgnoring.

Reply:
I like `changeReadyIgnoring(task *state.Task)` !